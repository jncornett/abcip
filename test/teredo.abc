d ( stack="eth:ip6:tcp" )
a ( ack, pay="abcdefghijklmnop" )

d ( stack="eth:ip4:udp:ip6:tcp" )
c ( 4:a=3797, b=3544 )
a ( ack, seq=16, pay="abcdefghijklmnop" )

d ( stack="eth:ip4:udp:ip6:udp" )
c ( 4:a=3797, b=3544 )
a ( pay="abcdefghijklmnop" )

d ( stack="eth:ip4:udp:ip6:icmp6" )
c ( 4:a=3797, b=3544 )
a ( pay="|53 aa 01 26|" )

d ( stack="eth:ip4:udp:ip6" )
c ( 4:a=3797, b=3544; )
a ( pay="abcdefg" )

d ( stack="eth:ip4:udp:ip6:udp" )
c ( 4:a=3797, b=3544 )
a ( head="|1 2 3|" )

