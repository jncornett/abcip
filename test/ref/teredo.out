IP6 (flowlabel 0x00001, hlim 64, next-header TCP (6) payload length: 36) ::ffff:10.1.2.3.48620 > ::ffff:10.9.8.7.8: Flags [.], cksum 0x7071 (correct), seq 0:16, ack 0, win 8192, length 16
	0x0000:  0209 0807 0605 0201 0203 0405 86dd 6000  ..............`.
	0x0010:  0001 0024 0640 0000 0000 0000 0000 0000  ...$.@..........
	0x0020:  ffff 0a01 0203 0000 0000 0000 0000 0000  ................
	0x0030:  ffff 0a09 0807 bdec 0008 0000 0000 0000  ................
	0x0040:  0000 5010 2000 7071 0000 6162 6364 6566  ..P...pq..abcdef
	0x0050:  6768 696a 6b6c 6d6e 6f70                 ghijklmnop
IP (tos 0x0, ttl 64, id 1, offset 0, flags [none], proto UDP (17), length 104)
    10.1.2.3.3797 > 10.9.8.7.3544: UDP, length 76
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0068 0001 0000 4011 5c71 0a01 0203 0a09  .h....@.\q......
	0x0020:  0807 0ed5 0dd8 0054 5e4a 6000 0001 0024  .......T^J`....$
	0x0030:  0640 0000 0000 0000 0000 0000 ffff 0a01  .@..............
	0x0040:  0205 0000 0000 0000 0000 0000 ffff 0a09  ................
	0x0050:  0807 bdec 0008 0000 0010 0000 0000 5010  ..............P.
	0x0060:  2000 705f 0000 6162 6364 6566 6768 696a  ..p_..abcdefghij
	0x0070:  6b6c 6d6e 6f70                           klmnop
IP (tos 0x0, ttl 64, id 1, offset 0, flags [none], proto UDP (17), length 92)
    10.1.2.3.3797 > 10.9.8.7.3544: UDP, length 64
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  005c 0001 0000 4011 5c7d 0a01 0203 0a09  .\....@.\}......
	0x0020:  0807 0ed5 0dd8 0048 536d 6000 0001 0018  .......HSm`.....
	0x0030:  1140 0000 0000 0000 0000 0000 ffff 0a01  .@..............
	0x0040:  0205 0000 0000 0000 0000 0000 ffff 0a09  ................
	0x0050:  0807 bdec 0008 0018 e068 6162 6364 6566  .........habcdef
	0x0060:  6768 696a 6b6c 6d6e 6f70                 ghijklmnop
IP (tos 0x0, ttl 64, id 1, offset 0, flags [none], proto UDP (17), length 80)
    10.1.2.3.3797 > 10.9.8.7.3544: UDP, length 52
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0050 0001 0000 4011 5c89 0a01 0203 0a09  .P....@.\.......
	0x0020:  0807 0ed5 0dd8 003c 2aae 6000 0001 000c  .......<*.`.....
	0x0030:  3a40 0000 0000 0000 0000 0000 ffff 0a01  :@..............
	0x0040:  0205 0000 0000 0000 0000 0000 ffff 0a09  ................
	0x0050:  0807 8000 0cd1 0001 0001 53aa 0126       ..........S..&
IP (tos 0x0, ttl 64, id 1, offset 0, flags [none], proto UDP (17), length 75)
    10.1.2.3.3797 > 10.9.8.7.3544: UDP, length 47
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  004b 0001 0000 4011 5c8e 0a01 0203 0a09  .K....@.\.......
	0x0020:  0807 0ed5 0dd8 0037 7a33 6000 0001 0007  .......7z3`.....
	0x0030:  3b40 0000 0000 0000 0000 0000 ffff 0a01  ;@..............
	0x0040:  0205 0000 0000 0000 0000 0000 ffff 0a09  ................
	0x0050:  0807 6162 6364 6566 67                   ..abcdefg
IP (tos 0x0, ttl 64, id 1, offset 0, flags [none], proto UDP (17), length 71)
    10.1.2.3.3797 > 10.9.8.7.3544: UDP, length 43
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0047 0001 0000 4011 5c92 0a01 0203 0a09  .G....@.\.......
	0x0020:  0807 0ed5 0dd8 0033 316b 6000 0001 0003  .......31k`.....
	0x0030:  1140 0000 0000 0000 0000 0000 ffff 0a01  .@..............
	0x0040:  0205 0000 0000 0000 0000 0000 ffff 0a09  ................
	0x0050:  0807 0102 03                             .....
